docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Keyboard Dongle
+sidebar_label: Keyboard Dongle
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+:::warning
+This implementation is experimental and subject to change.
+
+The information below is for keyboards consisting of a [board and a shield](../faq.md#why-boards-and-shields-why-not-just-keyboard). The setup for keyboards consisting exclusively of a [board](../faq.md#what-is-a-board) is not yet documented.
+:::
+
+A bluetooth dongle can be added to any keyboard running ZMK. The result is a [split keyboard](./split-keyboards.md) with the dongle as "central". There are a number of advantages to adding a dongle, but also some disadvantages:
+
+Benefits:
+
+- The "central" dongle results in increased battery life for the keyboard/former "central", as it is now a peripheral.
+- It is easier to connect to the host device.
+
+Disadvantages:
+
+- An extra microcontroller is needed.
+- The keyboard becomes unusable without the dongle.
+
+:::note
+Depending on how the dongle is used, there are some additional [latency considerations](./split-keyboards.md#latency-considerations) to keep in mind. 
+On average, the latency increase is around 3.75ms, in worst case scenario, the latency increase can be up to 7.5ms compared to a unibody keyboard.
+:::
+
+## Editing a Keyboard Definition to Support a Dongle
+
+You will modify the [files defining your keyboard](../config/index.md#shield-folder) to support a dongle. 
+The recommended approach to doing so (found below) allows you to easily enable a dongle, but it is disabled by default. 
+This is done to ensure that backwards compatibility is maintained for users who do not wish to use a dongle.
+
+### Shield Configuration Files
+
+Start by adding the dongle body to the shield configuration.
+
+```kconfig title="Kconfig.shield"
+config SHIELD_MY_KEYBOARD_DONGLE
+    def_bool $(shields_list_contains,my_keyboard_dongle)
+```
+
+Next, add the dongle configuration to your keyboard's `Kconfig.defconfig`.
+
+<Tabs
+defaultValue="unibody"
+values={[
+{label: 'Unibody Keyboard', value: 'unibody'},
+{label: 'Split Keyboard', value: 'split'},
+]}
+>
+<TabItem value="unibody">
+
+```kconfig title="Kconfig.defconfig"
+if SHIELD_MY_KEYBOARD_DONGLE
+
+config ZMK_KEYBOARD_NAME
+    default "My Board"
+
+config ZMK_SPLIT_ROLE_CENTRAL
+    default y
+
+config ZMK_SPLIT
+    default y
+
+
```

Comment:
In my opinion, this is optional and unrelated to using a dongle. I agree it may be helpful and perhaps merits a text reference to documentation of `BT_CTLR_TX_PWR_PLUS_8` such as https://zmk.dev/docs/troubleshooting/connection-issues